src/Toml/BiMap.hs

```diff
@@ -126,6 +134,12 @@ _TextToString = iso T.unpack T.pack
 _String :: BiMap AnyValue String
 _String = _Text >>> _TextToString
 
+_StringToShow :: (Show a, Read a) => BiMap String a
+_StringToShow = iso read show
```

Comment:
This is not actually an `iso` because you might not be able parse value of type `a` from `String`. So here you should use `prism` and `readMaybe`